.github/workflows/daily-cron_gen.yaml

```diff
@@ -59,13 +59,15 @@ jobs:
     - name: Configure Poetry
       run: poetry config virtualenvs.in-project true
     - name: Install packages needed for build
-      run: sudo apt-get install libkrb5-dev libsasl2-dev libpython3-dev g++ gcc
+      run: |-
+        sudo apt-get update
```

Comment:
This line is the only relevant one to get it running. Essentially, gh runners just need their apt cache refreshed.